bin/solc

```diff
@@ -1,72 +1,10 @@
-#!/bin/bash
+#!/usr/bin/env python3
+import os
+import sys
+from pathlib import Path
 
-function run_linux() {
-    export SSELECT_INSTALL_DIR="$HOME/.solc-select"
+sys.path.append(str(Path(os.path.dirname(os.path.realpath(__file__))).parent))
```

Comment:
The only problem with this is that people might decide to `cp bin/solc` somewhere else, in which case it will break the script. I'm fine leaving it in, but I'm not sure how useful it will be.

Reply:
I finally decided to remove it